### PR TITLE
some nbt work

### PR DESCRIPTION
**Pull request by eristhea**
aiming to complete n.m.nbt

### CODE REVIEW

**Comment by eristhea** [comment]
complete!

File: mappings/net/minecraft/nbt/NbtOps.mapping [hunk 1]
```diff
@@ -58,17 +69,89 @@ CLASS net/minecraft/unmapped/C_fybaywjo net/minecraft/nbt/NbtOps
...
+		METHOD <init> ([J)V
+			ARG 1 values
+	CLASS C_lrggpudd DelegatingListCollector
+		FIELD f_oknzgzfq INSTANCE Lnet/minecraft/unmapped/C_fybaywjo$C_lrggpudd;
+	CLASS C_nehsnwyp ListCollector
+		METHOD m_berihwml accept (Lnet/minecraft/unmapped/C_oivssbvb;)Lnet/minecraft/unmapped/C_fybaywjo$C_nehsnwyp;
+			ARG 1 element
+		METHOD m_fjtdnzmx accept (Ljava/lang/Iterable;)Lnet/minecraft/unmapped/C_fybaywjo$C_nehsnwyp;
+			ARG 1 elements
+		METHOD m_gxpktbdp acceptAll (Ljava/util/stream/Stream;)Lnet/minecraft/unmapped/C_fybaywjo$C_nehsnwyp;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/nbt/NbtOps.mapping]:
```suggestion
		METHOD m_gxpktbdp accept (Ljava/util/stream/Stream;)Lnet/minecraft/unmapped/C_fybaywjo$C_nehsnwyp;
			ARG 1 elements
```
or the `Iterable` one should also be `acceptAll`, but they should match

File: mappings/net/minecraft/nbt/NbtOps.mapping [hunk 2]
```diff
@@ -58,17 +69,89 @@ CLASS net/minecraft/unmapped/C_fybaywjo net/minecraft/nbt/NbtOps
...
+		METHOD get (Ljava/lang/String;)Ljava/lang/Object;
+			ARG 1 key
+	CLASS C_zcsovcmj WrappingListCollector
+		FIELD f_kuucpbgx result Lnet/minecraft/unmapped/C_hqhtfwlr;
+		METHOD <init> (Lit/unimi/dsi/fastutil/bytes/ByteArrayList;)V
+			ARG 1 byteList
+		METHOD <init> (Lit/unimi/dsi/fastutil/ints/IntArrayList;)V
+			ARG 1 intList
+		METHOD <init> (Lit/unimi/dsi/fastutil/longs/LongArrayList;)V
+			ARG 1 longList
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/nbt/NbtOps.mapping]:
```suggestion
			ARG 1 list
		METHOD <init> (Lit/unimi/dsi/fastutil/ints/IntArrayList;)V
			ARG 1 list
		METHOD <init> (Lit/unimi/dsi/fastutil/longs/LongArrayList;)V
			ARG 1 list
```

**Review by OroArmor — APPROVED** [review]

**Review by supersaiyansubtlety — COMMENTED** [review]
nitpicks

File: mappings/net/minecraft/nbt/NbtOps.mapping (hunk 1 above)
Comment (eristhea) [reply mappings/net/minecraft/nbt/NbtOps.mapping]: i prefer this, good catch

**Review by supersaiyansubtlety — APPROVED** [review]

**Pull request merged**